runtimes/manta/runtime/src/lib.rs

```diff
@@ -461,6 +467,7 @@ construct_runtime!(
         Balances: pallet_balances::{Module, Call, Storage, Config<T>, Event<T>},
         TransactionPayment: pallet_transaction_payment::{Module, Storage},
         Scheduler: pallet_scheduler::{Module, Call, Storage, Event<T>},
+		Utility: pallet_utility::{Module, Call, Event},
```

Comment:
nits: there seems to be a formatting issue here. can we align the code?

Reply:
Sorry, the indentation of my editor is tab, I'll replace it with space.